archaius2-core/src/main/java/com/netflix/archaius/config/AbstractConfig.java

```diff
@@ -128,8 +128,7 @@ public String getString(String key, String defaultValue) {
         if (value instanceof String) {
             return interpolator.create(getLookup()).resolve(value.toString());
         } else {
-            throw new UnsupportedOperationException(
-                    "Property values other than String not supported");
+            return value.toString();
```

Comment:
Is this the right contract here? If the type is not a string, should the return be value.toString(), or just defaultValue?